File: src/main/java/com/hiddewieringa/elevator/domain/elevator/event/ElevatorCreated.java
package com.hiddewieringa.elevator.domain.elevator.event;

import com.hiddewieringa.elevator.domain.elevator.model.ElevatorId;
import org.axonframework.commandhandling.TargetAggregateIdentifier;

public class ElevatorCreated {

    @TargetAggregateIdentifier
    private final ElevatorId elevatorId;

    public ElevatorCreated(ElevatorId elevatorId) {
        this.elevatorId = elevatorId;
    }

    public ElevatorId getElevatorId() {
        return elevatorId;
    }
}

File: src/main/java/com/hiddewieringa/elevator/domain/elevator/event/ElevatorCreated.kt
package com.hiddewieringa.elevator.domain.elevator.event

import com.hiddewieringa.elevator.domain.elevator.model.ElevatorId
import org.axonframework.commandhandling.TargetAggregateIdentifier

class ElevatorCreated(@field:TargetAggregateIdentifier
                      val elevatorId: ElevatorId)
